content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,397 @@
+for each container. It then kills the container with the highest score.
+
+This means that containers in low QoS pods that consume a large amount of memory
+relative to their scheduling requests are killed first.
+
+Unlike Pod eviction, if a container is OOM killed, the `kubelet` can restart it 
+based on its `RestartPolicy`.
+
+## Best practices
+
+The following sections describe best practices for eviction configuration.
+
+### Schedulable resources and eviction policies
+
+When you configure the kubelet with an eviction policy, you should make sure that
+the scheduler will not schedule pods if they will trigger eviction because they
+immediately induce memory pressure.
+
+Consider the following scenario:
+
+* Node memory capacity: `10Gi`
+* Operator wants to reserve 10% of memory capacity for system daemons (kernel, `kubelet`, etc.)
+* Operator wants to evict Pods at 95% memory utilization to reduce incidence of system OOM.
+
+For this to work, the kubelet is launched as follows:
+
+```
+--eviction-hard=memory.available<500Mi
+--system-reserved=memory=1.5Gi
+```
+
+In this configuration, the `--system-reserved` flag reserves `1.5Gi` of memory
+for the system, which is `10% of the total memory + the eviction threshold amount`. 
+
+The node can reach the eviction threshold if a pod is using more than its request,
+or if the system is using more than `1Gi` of memory, which makes the `memory.available`
+signal fall below `500Mi` and triggers the threshold. 
+
+### DaemonSet
+
+Pod Priority is a major factor in making eviction decisions. If you do not want
+the kubelet to evict pods that belong to a `DaemonSet`, give those pods a high
+enough `priorityClass` in the pod spec. You can also use a lower `priorityClass`
+or the default to only allow `DaemonSet` pods to run when there are enough 
+resources.
+
+## Deprecation of existing feature flags to reclaim disk 
+
+`kubelet` has been freeing up disk space on demand to keep the node stable.
+
+As disk based eviction matures, the following `kubelet` flags are marked for deprecation
+in favor of the simpler configuration supported around eviction.
+
+| Existing Flag                              | New Flag                                |
+| ------------------------------------------ | ----------------------------------------|
+| `--image-gc-high-threshold`                | `--eviction-hard` or `eviction-soft`    |
+| `--image-gc-low-threshold`                 | `--eviction-minimum-reclaim`            |
+| `--maximum-dead-containers`                | deprecated                              |
+| `--maximum-dead-containers-per-container`  | deprecated                              |
+| `--minimum-container-ttl-duration`         | deprecated                              |
+| `--low-diskspace-threshold-mb`             | `--eviction-hard` or `eviction-soft`    |
+| `--outofdisk-transition-frequency`         | `--eviction-pressure-transition-period` |
+
+## Known issues
```

Comment:
Have these issues been fixed since this page was last updated?

Reply:
I don't know who's best to confirm that. This advice does look stale. Try asking in `#sig-node` on the Kubernetes Slack.